src/lib/chain/offer.ts

```diff
@@ -334,52 +327,51 @@ export async function updateOffers(flags: OffersArgs) {
     );
   }
 
+  const [notFoundOffersInfo, offersInfo] = await getOffersInfo(offersToUpdate);
+
+  if (notFoundOffersInfo.length > 0) {
+    commandObj.warn(
+      commandObj.warn(
+        `Can't find offer info using the indexer:\n${notFoundOffersInfo
```

Comment:
User won't know what is an "indexer".

```
Cannot find the following offers:
  - name: id
  - name: id
```

Maybe also print full indexer output in DEBUG in that case. But could be clumsy from the code organisation perspective.